src/css/custom.css

```diff
@@ -22,6 +22,18 @@ body {
         rgba(0, 0, 0, 0.117647) 0px 1px 4px;
 }
 
+.search-bar {
+  margin-bottom: 24px;
+  width:100% !important;
+}
+
+/* images */
+.icon-search {
+  position: absolute;
```

Comment:
Wrapping it in a div as a container allows for using flex. I do not know if you can make the icon/textInput overlap with flex, though.

This webpage is a cool toy to experiment with flex https://cvan.io/flexboxin5/